runtime/binding-kafka/pom.xml

```diff
@@ -85,6 +85,10 @@
       <artifactId>jmh-generator-annprocess</artifactId>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>org.jboss.byteman</groupId>
+          <artifactId>byteman</artifactId>
+      </dependency>
```

Comment:
Please remove this dependency, it should be unnecessary. We only use it for TCP binding to force behavior of the socket APIs when simulating different network interactions.